locale/en/manageiq.po

```diff
@@ -2967,16 +2965,11 @@ msgstr ""
 msgid "Access Key"
 msgstr ""
 
-#: ../app/views/layouts/angular/_multi_auth_credentials.html.haml
+#: ../app/models/manageiq/providers/amazon/cloud_manager.rb
+#: ../app/models/manageiq/providers/amazon/container_manager.rb
 msgid "Access Key ID"
 msgstr ""
 
-#: ../app/views/layouts/angular/_multi_auth_credentials.html.haml
-msgid ""
-"Access Key ID and matching Secret Access Key fields are needed to perform veri"
-"fication of credentials"
-msgstr ""
-
```

Comment:
@GilbertCherrie can you double check the strings removed from the `angular*` paths?  I think they're from ManageIQ/manageiq-ui-classic#9020 and ManageIQ/manageiq-ui-classic#9021.  I'm not sure if the rest looks fine.